00_introduction/21_architecture_nodes.txt

```diff
@@ -33,18 +33,24 @@ available for these platforms:
 
 * Ubuntu 12.10 (quantal)
 
+* IBM AIX 5.3
+
+* IBM AIX 6.1
+
+* IBM AIX 7.1
+
 [TIP]
 
 [[install-on-windows, Install on Microsoft Windows]] 
 
-.Windows Nodes 
+.Windows and AIX Nodes
 
 ====
 
-Installing Rudder on Windows requires the commercial version of CFEngine (named Nova).
+Installing Rudder on Windows or AIX requires a commercial version the Rudder agent.
```

Comment:
Should we explain the difference between the Nova agent for Windows and AIX support?